src/pkg/packager/load/testdata/import/variables/zarf.yaml

```diff
@@ -7,11 +7,17 @@ constants:
 variables:
   - name: PARENT_VAR
     default: "default from parent"
+values:
+  files:
+    - parent-values.yaml
+  schema: "parent-values.schema.json"
```

Comment:
I don't yet see these files existing for testing purposes. Also should we perhaps instrument the testing separate from existing variable tests?

Reply:
considering scope of the unit testing - this seems intentional but maybe less than ideal.